docs/sources/slack/example-api-responses/sanitized/discovery-users-list.json

```diff
@@ -2,77 +2,52 @@
   "ok":true,
   "users":[
     {
-      "id":"p~BZcy5qhkyGyUTWR9hC_my2o6pHLKA4C8MHUHRy1K5TuugNC6SKYHB4eIKRZ4cFOg",
+      "id":"t~BZcy5qhkyGyUTWR9hC_my2o6pHLKA4C8MHUHRy1K5Ts",
       "deleted":false,
-      "color":"e7392d",
       "tz":"America/Los_Angeles",
       "tz_label":"Pacific Standard Time",
       "tz_offset":-28800,
       "profile":{
-        "email":"{\"domain\":\"domain.com\",\"hash\":\"hUnnRlSR8lUa37PCo-APXltncFZ3mPn3sgLmaI2pDA8\"}",
+        "email":"t~hUnnRlSR8lUa37PCo-APXltncFZ3mPn3sgLmaI2pDA8@domain.com",
         "team":"T02DDBANBM5"
       },
       "is_admin":true,
-      "is_owner":true,
-      "is_primary_owner":false,
-      "is_restricted":false,
-      "is_ultra_restricted":false,
       "is_bot":false,
-      "is_app_user":false,
       "updated":1635194243,
-      "is_email_confirmed":true,
-      "who_can_share_contact_card":"EVERYONE",
       "teams":[
         "T02DDBANBM5"
       ]
     },
     {
-      "id":"p~fyCnHbAdterJaFBMVnh6bo_qduSAt-k18eEgbYN_FqLZ-38BzAOn6weILvGYSJN4",
+      "id":"t~fyCnHbAdterJaFBMVnh6bo_qduSAt-k18eEgbYN_FqI",
       "deleted":false,
-      "color":"4bbe2e",
       "tz":"America/Los_Angeles",
       "tz_label":"Pacific Standard Time",
       "tz_offset":-28800,
       "profile":{
-        "email":"{\"domain\":\"domain.com\",\"hash\":\"DkRMkqDGNK3wQKF7dvYWOuRMDIJVwtlkVcLMbtFkNyc\"}",
+        "email":"t~DkRMkqDGNK3wQKF7dvYWOuRMDIJVwtlkVcLMbtFkNyc@domain.com",
         "team":"T02DDBANBM5"
       },
       "is_admin":false,
-      "is_owner":false,
-      "is_primary_owner":false,
-      "is_restricted":false,
-      "is_ultra_restricted":false,
       "is_bot":false,
-      "is_app_user":false,
       "updated":1631135209,
-      "is_email_confirmed":false,
-      "who_can_share_contact_card":"EVERYONE",
-      "is_invited_user":true,
       "teams":[
         "T02DDBANBM5"
       ]
     },
     {
-      "id":"p~mJlhl3xv52fNY6wyUps5mBYPCl1nVWHT442eQ7v-lJyurnRXe1xte4bUgJLOS0DP",
+      "id":"t~mJlhl3xv52fNY6wyUps5mBYPCl1nVWHT442eQ7v-lJw",
```

Comment:
:red_circle: this needs to be reversible for the new endpoint user.conversations, see https://github.com/Worklytics/psoxy/pull/836

Reply:
good catch; bug in rules generation wasn't respecting the `includeReversible` fix; corrected now.